docs/build/circuit-construction.ipynb

```diff
@@ -0,0 +1,486 @@
+      "text/plain": [
+       "<Figure size 203.885x367.889 with 1 Axes>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc_a = QuantumCircuit(4)\n",
+    "qc_a.x(0)\n",
+    "\n",
+    "qc_b = QuantumCircuit(2, name=\"qc_b\")\n",
+    "qc_b.y(0)\n",
+    "qc_b.z(1)\n",
+    "\n",
+    "# compose qubits (0, 1) of qc_a to qubits (1, 3) of qc_b respectively\n",
+    "combined = qc_a.compose(qc_b, qubits=[1, 3])\n",
+    "combined.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4529506-8397-4208-9cfd-31e1a1978741",
+   "metadata": {},
+   "source": [
+    "You may also want to compile circuits into instructions to keep your circuits organized. You can convert a circuit to an instruction using the [`to_instruction`](/api/qiskit/qiskit.circuit.QuantumCircuit#to_instruction) method, then append this to another circuit as you would any other instruction. The circuit drawn in the following cell is functionally equivalent to the circuit drawn in the previous cell."
```

Comment:
this info about the `to_instruction()` function feels a bit repetitive when you mentioned it already a few paragraphs earlier, I'd recommend just keeping one